source/agora/test/NetworkManager.d

```diff
@@ -76,16 +77,22 @@ unittest
 
             Block last_block;
             // make 20 blocks which have an invalid previous hash
-            foreach (idx; 0 .. 20)
+
+            // genesis case
+            if (!last_tx.length)
```

Comment:
Styling nit: when there's only a single statement then there is no need to add braces. So:

```D
if (!last_tx.length)
    last_tx = ...
```